internal/service/globalclusterconfig/resource_global_cluster_config_test.go

```diff
@@ -116,19 +103,8 @@ func TestAccClusterRSGlobalCluster_database(t *testing.T) {
 				),
 			},
 			{
-				Config: configWithDBConfig(&clusterInfo, customZoneUpdated),
-				Check: resource.ComposeTestCheckFunc(
-					checkExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "managed_namespaces.#", "5"),
-					resource.TestCheckResourceAttrSet(resourceName, "custom_zone_mappings.#"),
-					resource.TestCheckResourceAttrSet(resourceName, "custom_zone_mapping.%"),
-					resource.TestCheckResourceAttrSet(resourceName, "custom_zone_mapping.US"),
-					resource.TestCheckResourceAttrSet(resourceName, "custom_zone_mapping.IE"),
-					resource.TestCheckResourceAttrSet(resourceName, "custom_zone_mapping.DE"),
-					resource.TestCheckResourceAttrSet(resourceName, "custom_zone_mapping.JP"),
-					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
-					resource.TestCheckResourceAttr(resourceName, "cluster_name", clusterInfo.ClusterName),
-				),
+				Config:      configWithDBConfig(&clusterInfo, customZoneUpdated),
+				ExpectError: regexp.MustCompile("Updating a global cluster configuration resource is not allowed"),
```

Comment:
knit: testing the update error in one test should be enough, no need to test it twice